resources/views/tables/columns/server-entry-column.blade.php

```diff
@@ -1,65 +1,62 @@
+@php /** @var \App\Models\Server $server */ $server = $getRecord(); @endphp
```

Comment:
Getting claustrophobic here.

```suggestion
@php
    /** @var \App\Models\Server $server */
    $server = $getRecord();
@endphp
```